src/components/TableDialog.vue

```diff
@@ -140,13 +159,24 @@ export default Vue.extend({
         || !!this.fileUploadError
       );
     },
+
+    multinetTypes(): MultinetType[] {
+      return [
+        'label',
+        'category',
+        'number',
+        'date',
+      ];
+    },
```

Comment:
Why is this in computed? Nothing is changing here so I don't see a reason not to put it in `data`.

Reply:
This is a habit of mine when I want to make sure the value being specified cannot be changed by accident. It's sort of the `const` of Vue components.

Reply:
I think it's a bit of an anti-pattern, but I see the benefit of disallowing reassignment. I'm okay with this.

Reply:
Since this is our test case for transitioning to composition API, it will become moot since in that context we will just use a `const` declaration. So I'll let this stand for now with the understand that it will be "fixed" in that manner.